core/web/resolver/p2p.go

```diff
@@ -68,11 +70,19 @@ func (r *DeleteP2PKeySuccessResolver) Key() P2PKeyResolver {
 
 type DeleteP2PKeyPayloadResolver struct {
 	key p2pkey.KeyV2
-	err error
+	NotFoundErrorUnionType
 }
 
 func NewDeleteP2PKeyPayloadResolver(key p2pkey.KeyV2, err error) *DeleteP2PKeyPayloadResolver {
-	return &DeleteP2PKeyPayloadResolver{key, err}
+	var e NotFoundErrorUnionType
+
+	if err != nil {
```

Comment:
Similarly here.

Reply:
same reason as stated before on why this needs to happen